src/components/MenuBar/Menu/Menu.tsx

```diff
@@ -42,12 +54,21 @@ export default function Menu(props: { buttonClassName?: string }) {
           horizontal: 'center',
         }}
       >
+        <RecordingButton onClick={() => setMenuOpen(false)} setIsRecordingSnackbarOpen={setIsRecordingSnackbarOpen} />
+
+        <MenuItem onClick={() => setSettingsOpen(true)}>
+          <IconContainer>
+            <SettingsIcon />
+          </IconContainer>
+          <Typography variant="body1">Conference Settings</Typography>
```

Comment:
The button is labeled "Conference Settings", but the dialog box is "Audio and Video Settings". Should we make this consistent?

Reply:
We could - I'll check with Travis.

Reply:
I reverted this to 'Audio and Video Settings' per Travis.

Reply:
![Screen Shot 2021-01-12 at 10 34 08 AM](https://user-images.githubusercontent.com/22135968/104357194-b8c0da80-54c1-11eb-8892-c727e247bff0.png)
I see the code has been updated, but the deployed app is showing this ☝️ 
Do you have the latest deployed?